apps/backoffice-v2/.storybook/main.ts

```diff
@@ -18,5 +18,13 @@ const config: StorybookConfig = {
   docs: {
     autodocs: true,
   },
+  viteFinal: config => {
+    config.optimizeDeps = {
+      ...config.optimizeDeps,
+      include: ['@ballerine/ui'],
+    };
+
+    return config;
+  },
```

Comment:
Storybook no longer breaks in the backoffice if a component imports from @ballerine/ui